host_core/lib/host_core/control_interface/server.ex

```diff
@@ -130,14 +130,22 @@ defmodule HostCore.ControlInterface.Server do
 
   # Launch Actor
   # %{"actor_ref" => "wasmcloud.azurecr.io/echo:0.12.0", "host_id" => "Nxxxx"}
+  # %{"actor_ref" => "bindle://example.com/echo/0.12.0", "host_id" => "Nxxxx"}
   defp handle_request({"cmd", _host_id, "la"}, body, _reply_to) do
     with {:ok, start_actor_command} <- Jason.decode(body),
          true <-
            Map.has_key?(start_actor_command, "actor_ref") do
       Task.start(fn ->
-        case HostCore.Actors.ActorSupervisor.start_actor_from_oci(
-               start_actor_command["actor_ref"]
-             ) do
+        res =
+          if String.starts_with?(start_actor_command["actor_ref"], "bindle://") do
+            HostCore.Actors.ActorSupervisor.start_actor_from_bindle(
```

Comment:
cleanliness hint.. you can add the following to the top of the file so that you can remove all the module names from this call and just invoke `start_actor_from_bindle` (and the others) as though they were module-local:

```elixir
import HostCore.Actors.ActorSupervisor, only: [start_actor_from_bindle: 1, start_actor_from_oci:1]
import HostCore.Providers.ProviderSupervisor, only: [start_provider_from_bindle:1, start_provider_From_oci:1]
```